src/backend/expungeservice/form_filling.py

```diff
@@ -61,6 +61,15 @@
     "yamhill": "District Attorney - 535 NE 5th St #117 - McMinnville, OR 97128",
 }
 
+T = TypeVar("T")
+
+
+def join_dates_or_strings(arr=List[T], connector="; ", date_format="%b %-d, %Y") -> str:
```

Comment:
```suggestion
def join_dates_or_strings(arr: List[T], connector="; ", date_format="%b %-d, %Y") -> str:
```

Are you sure you don't mean this? Is it a default value of List[T]?

Reply:
Btw are you aware of https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments? Probably not relevant here but was reminded of it.

Reply:
Ok on a closer look, I actually think not having default arguments is the way to go here. Just pass it in. Explicit is better than implicit and since you only use it twice anyways, I don't think the default arguments are worth it - yet.
```
def join_dates_or_strings(arr, connector, date_format) -> str:
```

Reply:
Good catch. 👍🏽  Definitely unintended. I'm surprised the checkers didn't flag that.

I just fixed that and removed the defaults. Also, I replaced the generic type with something more specific since the function isn't that generic.

Really appreciate the time you take on the code review. I'm going to grab a copy of that book. Looks like good stuff.